docs/public/troubleshooting.md

```diff
@@ -190,6 +190,19 @@ There are two ways to increase the limit value:
 
   Don't forget to restart Consul servers to apply made changes.
 
+## Failed to get log
+
+Problem with error log "agent.server.raft: failed to get log" allows any action:
+  
+  1. Check backups in backup daemon, likes "/listbackups", "/listbackups/ID". If status equal "Successful" and if last backup is old, then we nees to
+     data loss
+  2. If you have a time to start pods, then connect to pod and delete raft.db file on all pods. If you don't have a time, then connect to kubernetes node in directory with consul PV and delete this file.
```

Comment:
I suggest to split into child list and remove "If you have a time": 
Resolving issue with raft.db || Removing raft.db, you can follow one of the next actions: 

1. Removing from pod directly: connecto to a pod and remove file: 
2. Removing file from PV

Also I think it's require more details about what each action has Pros/Cons. So reader will have a quite a lot information for making decision.